controllers/utils.go

```diff
@@ -77,3 +79,14 @@ func containsString(slice []string, s string) bool {
 	}
 	return false
 }
+
+func stripSensitiveData(err error) error {
```

Comment:
Not important for this PR, as we are foreseeing that it will be refactored soon (or handled differently), but in cases like this I would pick `maskSensitiveData` over the current function name.

As a strip would literally take all data out, instead of replacing (= masking) it.